ui/aura/mus/window_tree_host_mus_init_params.cc

```diff
@@ -36,6 +36,10 @@ WindowTreeHostMusInitParams CreateInitParamsForTopLevel(
       static_cast<WindowTreeHostMusDelegate*>(window_tree_client)
           ->CreateWindowPortForTopLevel(&properties);
   params.properties = std::move(properties);
+#if defined(USE_OZONE) && defined(OS_LINUX) && !defined(OS_CHROMEOS)
```

Comment:
c00l! It works. But please remove the duplication here - https://github.com/Igalia/chromium/blob/ozone-wayland-dev/chrome/browser/ui/views/frame/browser_frame_mus.cc#L79

Because browser_frame_mus uses the same CreateInitParamsForTopLevel first. - https://github.com/Igalia/chromium/blob/ozone-wayland-dev/chrome/browser/ui/views/frame/browser_frame_mus.cc#L75

Reply:
done